packages/utils/ts-utils/package.json

```diff
@@ -22,9 +22,9 @@
     "nullthrows": "^1.1.1"
   },
   "devDependencies": {
-    "typescript": ">=3.0.0"
+    "typescript": "~4.7"
   },
   "peerDependencies": {
-    "typescript": ">=3.0.0"
+    "typescript": "3.0.0 - 4.7"
```

Comment:
I personally would immediately discontinue support for `typesscript@3`. I am heavily doubt that `typescript@3` is really compatible with current `parcel`. There are no tests run with `typescript@3` so I would say: Nobody knows for sure. Instead of creating a test matrix with older typescript versions, I would rather just discontinue support for `typescript@3`.